src/utils/sqsProcess.ts

```diff
@@ -0,0 +1,10 @@
+import DynamoDB from "aws-sdk/clients/dynamodb";
```

Comment:
With this function being identical to the sqsProcess in atf-report-gen, is this a potential candidate to go into the cvs-microservice-common repo?